tensorboard/writer/BUILD

```diff
@@ -0,0 +1,42 @@
+# Description:
+# TensorBoard, a dashboard for investigating TensorFlow
+
+package(default_visibility = ["//tensorboard:internal"])
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+py_library(
+    name = "writer",
+    srcs = glob([
+        "*.py",
```

Comment:
Can we do an explicit  list here? Looks like it would be:

```
    srcs = [
        "__init__.py",
        "crc32c.py",
        "event_file_writer.py",
        "record_writer.py",
    ],